### PR TITLE
Demo with typeahead.js

### PR DESCRIPTION
**Pull request by GreatWizard**
:warning: This PR is above jquery-autocomplete! :warning: 

Issue: same as https://github.com/novafloss/django-agnocomplete/pull/10

### CODE REVIEW

File: demo/tests/test_demo_views.py [hunk 1]
```diff
@@ -117,3 +117,14 @@ def test_widgets(self):
...
+
+class JSDemoViews(TestCase):
+
+    def test_selectize(self):
+        response = self.client.get(reverse('selectize'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_jquery_autocomplete(self):
+        response = self.client.get(reverse('jquery-autocomplete'))
+        self.assertEqual(response.status_code, 200)
```
Comment (brunobord) [inline demo/tests/test_demo_views.py:146]: could you please add tests for typeahead view?

Comment (GreatWizard) [inline demo/tests/test_demo_views.py:146]: Done :)

**Comment by brunobord** [comment]
it looks a bit ugly here: is it normal or can we improve the look'n'feel?

typeahead

**Comment by brunobord** [comment]
all right, it looks like typeahead is ugly because of the utf8 chars in fixtures. We'll try to introspect into that at some point.

**Pull request merged**